Files/Helpers/RegistryReader.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Win32;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Files.Helpers
+{
+    class RegistryReader
+    {
+        private bool IsAdministrator()
+        {
+            return (new WindowsPrincipal(WindowsIdentity.GetCurrent()))
```

Comment:
Are Universal Windows apps currently allowed to access WindowsPrincipal?

Reply:
I am seeing it as supported under UWP: https://stackoverflow.com/questions/36713824/how-to-get-group-infolike-guest-admin-etc-of-current-login-user-in-uwp-windo. Let me know if this is a concern.

Reply:
I am seeing it as supported under UWP: https://stackoverflow.com/questions/36713824/how-to-get-group-infolike-guest-admin-etc-of-current-login-user-in-uwp-windo. Let me know if this is a concern.

Reply:
I am seeing it as supported under UWP: https://stackoverflow.com/questions/36713824/how-to-get-group-infolike-guest-admin-etc-of-current-login-user-in-uwp-windo. Let me know if this is a concern.